x-pack/test/functional/apps/observability/alerts/index.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import querystring from 'querystring';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+// Based on the x-pack/test/functional/es_archives/observability/alerts archive.
+const DATE_WITH_DATA = {
+  rangeFrom: '2021-08-23T13:36:22.109Z',
+  rangeTo: '2021-08-25T13:36:22.109Z',
+};
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+
+  describe('Observability alerts', function () {
+    this.tags('includeFirefox');
+
+    const pageObjects = getPageObjects(['common']);
+    const testSubjects = getService('testSubjects');
+
+    before(async () => {
```

Comment:
Is this like beforeEach or beforeAll?

I spoke with Dario yesterday and he mentioned that within APM they have a setup that loads one dataset for all of their functional tests (rather than per suite), is that something we want to copy? 
The downside is that the tests risk becoming more coupled but the gain is speed due to less data loading.

Reply:
`before` could be considered the same as `beforeAll`, it'll run once for the `describe` block.

> is that something we want to copy?

It's a good question. APM's setup is fairly custom (if this is regarding the tests at `test/apm_api_integration`) where they have their own config, helpers etc. So we could potentially do something similar, but it wouldn't really follow the expected format of tests at `test/functional/apps` 🤔

Reply:
Might be better to stick to conventions then and see how the speed of the test framework may be improved instead.